sqlalchemy_pytds/connector.py

```diff
@@ -96,8 +96,7 @@ def create_connect_args(self, url):
             if param in keys:
                 connect_args[param] = keys.pop(param)
 
-        connect_args["dsn"] = connect_args["host"]
-        del connect_args["host"]
+        connect_args["dsn"] = connect_args.pop("host", "localhost")
```

Comment:
there may be a better default better than `localhost`, feel free to adjust accordingly.